### PR TITLE
Add boards and playbooks guide for PostgreSQL migration

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
This PR adds the necessary documentation to migrate boards and playbooks schemas to PostgreSQL from MySQL.

Note that this is a companion PR for https://github.com/mattermost/docs/pull/6596 and needs to be merged after that one gets in. But we can start discussing on the script at this point.

### CODE REVIEW

**Review by nab-77 — APPROVED** [review]
Great clarification @isacikgoz

**Comment by nab-77** [comment]
@cwarnermm I think we are ready to publish this now

**Comment by cwarnermm** [comment]
@nab-77 - Where would you like this page positioned in the navigation pane? That's the last step of this docs PR (and one I can complete once we know where it should live).

**Comment by nab-77** [comment]
So I made a mistake earlier. The one to publish first is https://github.com/mattermost/docs/pull/6596.

So your point about nav location: https://docs.mattermost.com/guides/administration.html#self-hosted-administration (for now).

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

File: source/guides/postgres-migration.rst [hunk 1]
```diff
@@ -192,6 +194,181 @@ For our case we can simply run the following command:
...
+       $$ CREATE INDEX IF NOT EXISTS ir_playbookmember_playbookid on {{ .source_schema }}.IR_PlaybookMember(PlaybookId); $$,
+       $$ ALTER SCHEMA {{ .source_schema }} RENAME TO public; $$,
+       $$ SELECT pg_catalog.set_config('search_path', '"$user", public', false); $$,
+       $$ ALTER USER {{ .pg_user }} SET SEARCH_PATH TO 'public'; $$;
+
+.. code:: bash
+
+   pgLoader playbooks.load > playbooks_migration.log
+
+Boards
```
Comment (cwarnermm) [inline source/guides/postgres-migration.rst:312]:
```suggestion
Focalboard
```

File: source/guides/postgres-migration.rst [hunk 2]
```diff
@@ -192,6 +194,181 @@ For our case we can simply run the following command:
...
+~~~~~~
+
+As of ``v9.0`` Boards will transition to being fully community supported. Hence this guide covers only the version ``v7.10.x`` of the schema. `Official announcement <https://docs.mattermost.com/install/deprecated-features.html#mattermost-server-v9-0-0>`__.
```
Comment (cwarnermm) [inline source/guides/postgres-migration.rst:315]:
```suggestion
As of ``v9.0`` Boards will transition to being fully community supported as the Focalboard plugin. Hence this guide covers only the version ``v7.10.x`` of the schema. `Official announcement <https://docs.mattermost.com/install/deprecated-features.html#mattermost-server-v9-0-0>`__.
```

**Review by cwarnermm — APPROVED** [review]
Thank you, @isacikgoz! I've provided suggestions inline. Approving for expediency.

**Pull request merged**